_posts/blog/2016-09-27-droidcon_nyc.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "Droidcon NYC 2016"
+description: Deep knowledge from the best Android developers in the world.
+modified:
+categories: blog
+author: florina_muntenescu
+excerpt: Deep knowledge was shared by some of the best Android developers in the world at Droidcon NYC 2016. Here are some of the things we learned.
+tags: [Conference, Architecture, MVC, MVP, MVVM, RecyclerView, ConstraintLayout]
+date: 2016-09-27T10:39:55-04:00
+---
+I've just got back from Droidcon NYC were I gave a talk on <a href="http://www.slideshare.net/FlorinaMuntenescu/a-journey-through-mv-wonderland-66570666">MV* architecture</a>, a topic we also started on the blog. While there, but also via the live stream, we got the chance to see more than sixty top developers from some of the most renowned companies, sharing their knowledge on the most interesting topics in Android today. Talks on subjects like performance, Material Design, architecture, deep understanding of different types of Views or Layouts and war stories were all gathered at Droidcon NYC in two intense days. Here are some of the talks, tips or libraries that we considered the most interesting. Some of the videos are already available on <a href="http://droidcon.nyc/lineup">Droidcon NYC's website</a> for ticket holders.
+Until all videos are posted publicly, we're providing the links to the slides, where available.
+
+
+For an app like upday, that is pre-installed on Samsung devices, the **cold start time** is very important. Vikram Bodicherla from Yahoo gave some good tips about how to measure the startup time: using `SystemClock.elapsedRealTime()`; how to analyze the stack trace but also on what you can do to improve it: apply the *Most important pixel first* rule - display the most important area first.
```

Comment:
You could make `Vikram Bodicherla` a link to his twitter `https://twitter.com/vikramb0`